src/layouts/EditHomepage/EditHomepage.jsx

```diff
@@ -793,6 +882,27 @@ const EditHomepage = ({ match }) => {
               )
             }
             setHomepageState(intermediateHomepageState)
+          } else if (val.infocols) {
+            // Create 3 infoboxes by default
+            const parentId =
```

Comment:
could i get some clarity on why the last element is the parent?

Reply:
this happens within the create handler. so at the point of creation of a new block, this is being added to the last index of the sections array
hence, for the added element, if it is infocols, the parentId of this will be the last index of the sections array

Reply:
same is being done for textcards as well

Reply:
gotcha - no issue with this, just that the variable `parentId` is abit confusing. could we just add this comment there

Reply:
added as comment

Reply:
wait if it is handed over to the last element of the array, what happens during a reorder of the elements? if infocols and text cards are reordered, does this mean that n && n +1 swaps places with m && m + 1 respectively, where n in the index of textcard and m is the index of infocol?

Reply:
If u see src/hooks/useDrag.tsx, we will need to re-compute the state of the specific section (updatePositions) - so we do a shifting of the indices from source index to the new destination index

Reply:
lets huddle over this to speed up review

Reply:
Is this needed because there could be more than one infocol?

Reply:
yea, hence we need to track to which section the infobox belongs to